frontend/catalyst/pipelines.py

```diff
@@ -45,12 +169,13 @@
         # But qnodes targetting other backends may choose to lower
         # this into something else.
         "inline-nested-module",
-    ],
-)
+    ]
+    return enforce_runtime_invariants
+
 
-HLO_LOWERING_PASS = (
-    "HLOLoweringPass",
-    [
+def get_hlo_lowering_stage(_options: CompileOptions) -> List[str]:
```

Comment:
Forgive my Python: so` _options` is used to denote an argument that is not used at the moment. We keep it to be consistent with similar functions actually using the argument, and because we might use the argument in the future. Is that correct?

Reply:
Correct. Variables that start with the underscore character are an indication to pylint (or other linter tools) that the variable is meant for internal use and pylint ignores it. I think it is a good idea to pass the options to these functions as I see these functions typed and I want them to have the same type.